tests/testCLI.js

```diff
@@ -0,0 +1,87 @@
+const assert = require('assert');
+const chai = require('chai');
+const sinon = require('sinon');
+const util = require('util');
+
+const {cli, utils} = require('../index');
+
+
+describe('cli function', () => {
+    afterEach(function () {
+        sinon.restore();
+    });
+
+    it('should ensure log token is set if logs are enabled', () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu'])
+        }).to.throw('Logs enabled but no log token!');
+    });
+
+    it('should ensure stats token is set if stats are enabled', () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu', '--logstoken', 'asd'])
+        }).to.throw('Stats enabled but no stats token!');
+    });
+
+    it('should ensure events token is set if events are enabled', () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu', '--logstoken', 'asd', '--statstoken', 'das good'])
+        }).to.throw('Events enabled but no events token!');
+    });
+
+
+    it("should not crash if log token isn't set if logs are disabled", () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu', '--no-logs'])
+        }).to.not.throw('Logs enabled but no log token!');
+    });
+
+    it("should not crash if stats token isn't set if stats are disabled", () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu', '--no-logs', '--no-stats'])
+        }).to.not.throw('Stats enabled but no stats token!');
+    });
+
+    it("should not crash events token isn't set if events are disabled", () => {
+        chai.expect(() => {
+            cli(['node', 'index.js', '-r', 'eu', '--no-logs', '--no-stats', '--no-events'])
```

Comment:
it'd be nice to make sure these tests can't actually hit the real production instances. not a disaster if they do, obviously. but i think we support datahub so we _could_ point it at `localhost`, i think?

Reply:
afaik none of these tests get to actually sending data.

They all error out one way or another since there's no happy path test here

Also no support for datahub in this repo